pulse_actions/worker.py

```diff
@@ -49,13 +50,16 @@ def run_pulse(exchange, topic, event_handler, topic_base, dry_run=True):
     def handler_with_dry_run(data, message):
         return event_handler(data, message, dry_run)
 
-    pulse = PulseConsumer(exchange,
+    pulse = PulseConsumer(exchanges,
                           callback=handler_with_dry_run,
                           **pulse_args)
-    LOG.info('Listening on %s, with topic %s', exchange, topic)
+    LOG.info('Listening on %s, with topic %s', exchanges, topics)
 
     while True:
-        pulse.listen()
+        try:
+            pulse.listen()
+        except Exception as e:
+            LOG.info("Error: %s" % e)
 
```

Comment:
@vaibhavmagarwal  that is better then what I was asking. Good idea to add "Error:" to the beginning of the message, that will make it easier to search for (I was just logging the plain message and it was hard to search for).